lib/resque/scheduler/lua.rb

```diff
@@ -0,0 +1,43 @@
+module Resque
+  module Scheduler
+    module Lua
+      extend self
+
+      def self.zpop(key, *args)
+        evalsha(:zpop, [key], args)
```

Comment:
We require all arguments in the lua script, correct? Should we enumerate them all here then to prevent wrong usage?

Reply:
hmm. Sure, I don't see any downside and we get better errors.